docs/core/diagnostics/observability-with-otel.md

```diff
@@ -0,0 +1,422 @@
+---
+title: .NET Observability with OpenTelemetry
+description: An introduction to observing .NET apps with OpenTelemetry
+ms.date: 6/14/2023
+---
+
+# .NET Observability with OpenTelemetry
+
+When running a distributed application, operators want to know how well the application is performing, and to try to detect potential problems when they are still small, before they become larger. One way that they can do this is by monitoring specific instrumentation emitted by the application, and looking for changes in metrics and logs.
+
+## What is Observability
+
+Observability in the context of a distributed system is the ability to monitor and analyze telemetry about the state of each component in the system, to be able to observe changes in the performance, and diagnose why those changes occur. Unlike debugging which is invasive and can affect the operation of the application, observability is intended to be transparent to the primary operation and have a small enough performance impact that it can be used continuously.
+
+Observability is commonly done using a combination of:
+
+- Logs - recording individual operations, such as an incoming request, a failure in a specific component, or an order having been placed.
+- Metrics - measuring counters and gauges such as count of completed requests, number of active requests, number of widgets that have been sold,  or a histogram of the request latency.
```

Comment:
should only be one "or" right?

Reply:
I was thinking of it as:

- Number of completed requests, active requests, or widgets that have been sold
Or
- A histogram of the request latency

Reply:
This is where my grammar knowlege for use of , and ; in lists is lacking. based on https://www.merriam-webster.com/words-at-play/a-guide-to-using-semicolons how about:

- Metrics - measure counters and gauges, such as number of completed requests, active requests, or widgets that have been sold; or a histogram of the request latency.

Reply:
Based on this in the link you shared, I like what you suggested!

"A semicolon can also replace a comma between two clauses that are joined by a coordinating conjunction like and in cases where the sentence might otherwise be confusing—for example, because of particularly long clauses, or the presence of other commas."